src/cpp/middleware/FastEntities.cpp

```diff
@@ -267,7 +280,10 @@ void FastDataReader::onSubscriptionMatched(fastrtps::Subscriber*, fastrtps::rtps
 
 void FastDataReader::onNewDataMessage(fastrtps::Subscriber *)
 {
-    on_new_data_cb_();
+    if (nullptr != on_new_data_cb_)
+    {
+        on_new_data_cb_();
```

Comment:
Only for curiosity, there is a pretty shortcut for this pattern: 
```c
on_new_data_cb && on_new_data_cb_()
```

Reply:
xD, I'm not sure but it will be nice.